### PR TITLE
Re-structured the main page

### PR DESCRIPTION
**Pull request by CarlSargunar**
I shuffled it a little to put news below sponsors, and code of c…onduct directly linked. To my eye the news should be top, and it links directly to the schedle, and I the green section below has a link to the code of conduct

Purely subjective, but I think it makes the main page a little easier to read

### CODE REVIEW

**Review by slang25 — APPROVED** [review]
Yeah I agree this is better 👍

**Comment by dynamictulip** [comment]
It's great to have the latest update more prominent but a new visitor to the page (who isn't familiar with DDD South West) has to scroll a long way down before they find out that we're a conference (opposed to say a meetup). Can we move the intro text back up the page?

**Comment by CarlSargunar** [comment]
@dynamictulip I've tweaked it a bit based on your points - changing the top banner. 

Still playing around with the layout but open to feedback

File: src/lib/Carousel.svelte [hunk 1]
```diff
@@ -22,7 +22,8 @@
 {#key carouselImage}
 	<div class="header" style="background-image: url({carouselImage.src});">
 		<div class="details-container shadow">
-			<span class="name">DDD South West</span><br />
+			
+			<h1 class="name">DDD South West Conference</h1><br />
```
Comment (dynamictulip) [inline src/lib/Carousel.svelte:26]: This change to be an `h1` is great! Unfortunately it seems to be causing the vertical alignment of the text to change and the overall picture box to become larger

Comment (dynamictulip) [inline src/lib/Carousel.svelte:26]: I don't think we should add "Conference" to the title as I think this should match what we call ourselves - i.e. "DDD South West"

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -29,30 +29,32 @@
...
+
 <div class="introduction primary-bg">
 	<div class="section">
-		<p class="text-center">
-			DDD South West is an inclusive, non-profit developer conference organised by the community,
-			for the community.
+		<h2>Latest updates</h2>
+		<p>
+			<a href="/schedule">Schedule is announced!!</a> Time for last second tweaks to your talks, test
+			your demos and get ready for the big day!!
```
Comment (dynamictulip) [inline src/routes/+page.svelte:41]: On larger screen resolutions, this text is bigger than the associated `h2` which looks odd. Perhaps we could shrink it a little?

File: src/lib/Carousel.svelte (hunk 1 above)
Comment (dynamictulip) [reply src/lib/Carousel.svelte:26]: I'm not sure where we should put it though if we remove the tag line 😅

**Comment by CarlSargunar** [comment]
We should close this now - I don't think there's anything in this branch we need anymore @dynamictulip ?

**Comment by dynamictulip** [comment]
> We should close this now - I don't think there's anything in this branch we need anymore @dynamictulip ?

I agree 👍

**Pull request closed without merging**